spec/features/features_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'order'
+require 'menu'
+require 'calculate_bill'
+require 'message'
+
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+
+describe Order do
+  subject(:order){described_class.new}
+  
+  it 'starts a new order' do
+   order
+  end
+
+  describe '#show_menu' do
+    it 'shows the list of available dishes' do
+    order.show_menu
+    end
+  end
+
+# As a customer
+# So that I can order the meal I want
+# I would like to be able to select some number of 
+# several available dishes
+  
+  describe '#choose' do
+   it 'records the quantity as well' do
+    order.choose("Margherita", 2)
+  end
```

Comment:
Lint/BlockAlignment: `end` at 31, 2 is not aligned with `it 'records the quantity as well' do` at 29, 3.